Scripts/RemediationScripts/Remediate-SetServiceBusMinReqTLSVersion.ps1

```diff
@@ -0,0 +1,813 @@
+            return
+        }
+    }
+    else {
+        Write-Host "[Step 1 of 3] Validate the user..." 
+        Write-Host $([Constants]::SingleDashLine)
+    } 
+
+    # Connect to Azure account
+    $context = Get-AzContext
+
+    if ([String]::IsNullOrWhiteSpace($context)) {
+        Write-Host $([Constants]::SingleDashLine)
+        Write-Host "Connecting to Azure account..."
+        Connect-AzAccount -Subscription $SubscriptionId -ErrorAction Stop | Out-Null
+        Write-Host "Connected to Azure account." -ForegroundColor $([Constants]::MessageType.Update)
+    }
+    else {
+        # Setting up context for the current Subscription.
+        $context = Set-AzContext -SubscriptionId $SubscriptionId -ErrorAction Stop
+    }
+
+    Write-Host $([Constants]::SingleDashLine)
+    Write-Host "Subscription Name: [$($context.Subscription.Name)]"
+    Write-Host "Subscription ID: [$($context.Subscription.SubscriptionId)]"
+    Write-Host "Account Name: [$($context.Account.Id)]"
+    Write-Host "Account Type: [$($context.Account.Type)]"
+    Write-Host $([Constants]::SingleDashLine)
+
+    # Note about the required access required for remediation
+
+    Write-Host "To set secure TLS version for Azure Service Bus Namespace(s) in the Subscription, Contributor on Subscription or Resource Group or Azure Data Owner role on Service Bus Namespace(s) is required." -ForegroundColor $([Constants]::MessageType.Warning)
+
+    Write-Host $([Constants]::SingleDashLine)
+    Write-Host "[Step 2 of 3] Prepare to fetch all Azure Service Bus Namespace(s)"
+    Write-Host $([Constants]::SingleDashLine)
+
+    if (-not (Test-Path -Path $RollbackFilePath)) {
+        Write-Host "Input file:  [$($RollbackFilePath)] not found. Exiting..." -ForegroundColor $([Constants]::MessageType.Error)
+        Write-Host $([Constants]::DoubleDashLine)	
+        return
+    }
+
+    Write-Host "Fetching all Azure Service Bus Namespace(s) from" -NoNewline
+    Write-Host " [$($RollbackFilePath)]..." -ForegroundColor $([Constants]::MessageType.Info)
+    Write-Host $([Constants]::SingleDashLine)
+    $serviceBusNamespacesDetails = Import-Csv -LiteralPath $RollbackFilePath
+
+    $validServiceBusNamespacesDetails = $serviceBusNamespacesDetails | Where-Object { ![String]::IsNullOrWhiteSpace($_.ResourceId) -and ![String]::IsNullOrWhiteSpace($_.ResourceGroupName) -and ![String]::IsNullOrWhiteSpace($_.ResourceName) }
+
+    $validServiceBusNamespacesDetailsCount = $(($validServiceBusNamespacesDetails | Measure-Object).Count)
+
+    if ($validServiceBusNamespacesDetailsCount -eq 0) {
+        Write-Host "No Azure Service Bus Namespace(s) found. Exiting..." -ForegroundColor $([Constants]::MessageType.Warning)
+        Write-Host $([Constants]::DoubleDashLine)	
+        return
+    }
+
+    Write-Host "Found [$($validServiceBusNamespacesDetailsCount)] Azure Service Bus Namespace(s)." -ForegroundColor $([Constants]::MessageType.Update)
+
```

Comment:
a line space would be good here between table and this statement,

Reply:
adding a single dash line